src/ts/backend/save/saveFileData.ts

```diff
@@ -25,11 +25,9 @@ import { Result } from "@/utils/types";
 import { SaveLoadingError } from "./saveLoadingError";
 import { safeParseSaveV2, SaveSchemaV2 } from "./v2/saveV2";
 
-export const SaveSchema = SaveSchemaV2.extend({
-    thumbnail: z.string().optional(),
-});
+export const SaveSchema = SaveSchemaV2;
 
-export type Save = z.infer<typeof SaveSchema>;
+export type Save = z.infer<typeof SaveSchemaV2>;
```

Comment:
There is no need to change that. If we get a SaveSchemaV3 one day, then chaning to  `SaveSchema = SaveSchemaV3` should be the only change needed.

If you change this line then we no longer have a single source of truth